### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Naylin15**
## Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: [Conventional Commit](https://conventionalcommits.org/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

No content on the project site
Issue # N/A

## What is the new behavior?
New content in the index of the gh-pages

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Hi, @ajsb85 

This PR adds new content to the project site

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- initsession() 
+- killsession()
+- getMyProfiles()
+- getActiveProfile() 
+- changeActiveProfile()
+- getMyEntities()
+- getActiveEntities()
+- changeActiveEntities()
+- getFullSession()
+- getAnTtem()
```
Comment (rafaelje) [inline index.md]: Hi @Naylin15 please change this method name getAnItem by getItem() and add this 2 more:

- recoveryPassword()
- resetPassword()

**Review by rafaelje — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (Naylin15) [reply index.md]: Right away

**Review by Naylin15 — COMMENTED** [review]
Changes made 👍

**Review by rafaelje — APPROVED** [review]
Reviewed 👍

**Pull request merged**